quinn-udp/src/lib.rs

```diff
@@ -32,14 +32,6 @@ mod imp;
 
 pub use imp::UdpSocketState;
 
-/// Whether transmitted datagrams might get fragmented by the IP layer
-///
-/// Returns `false` on targets which employ e.g. the `IPV6_DONTFRAG` socket option.
-#[inline]
-pub fn may_fragment() -> bool {
```

Comment:
As suggested in @Ralith's comment, please split this change (moving the `may_fragment()` function into the `UdpSocketState` impl) into a separate commit from the changes that actually make the answer stateful.

Reply:
done

Reply:
My intent with this comment was to have the change to move `may_fragment()` part of `UdpSocketState` in the first commit. Currently it seems the first commit will just fail to compile, which is definitely not the intention.

Reply:
got it